src/tools/crossgen2/ILCompiler.ReadyToRun/Compiler/DependencyAnalysis/ReadyToRunSymbolNodeFactory.cs

```diff
@@ -164,7 +236,7 @@ private ISymbolNode CreateGCStaticBaseHelper(TypeDesc type, SignatureContext sig
                 _codegenNodeFactory,
                 _codegenNodeFactory.HelperImports,
                 ILCompiler.ReadyToRunHelper.DelayLoad_Helper,
-                _codegenNodeFactory.TypeSignature(ReadyToRunFixupKind.READYTORUN_FIXUP_StaticBaseGC, type, signatureContext));
+                _codegenNodeFactory.TypeSignature(ReadyToRunFixupKind.READYTORUN_FIXUP_StaticBaseGC, type));
```

Comment:
This doesn't look right. How come we can afford to drop the signature context on the floor? Have you tried to run CoreCLR tests in release / large version bubble mode with your change? I would expect them to be crashing left and right because of this. GetSignatureContext just returns the input module context but that's not usable everywhere; in particular, all these places truly need to pass the signature context information along, probably as a new member in TypeFixupKey.